source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
I think unary looks a little strange with a space, but maybe that's just me. Either way, be consistent.